src/config/base.rs

```diff
@@ -97,26 +106,40 @@ impl Base {
 
     fn s3_panic_if_missing_keys() -> Uploader {
         Uploader::S3 {
-            bucket: s3::Bucket::new(
+            bucket: Box::new(s3::Bucket::new(
                 env("S3_BUCKET"),
                 dotenv::var("S3_REGION").ok(),
                 env("S3_ACCESS_KEY"),
                 env("S3_SECRET_KEY"),
                 "https",
-            ),
+            )),
+            index_bucket: Box::new(s3::Bucket::new(
+                env("S3_INDEX_BUCKET"),
```

Comment:
quoting from the env sample file:

> You can leave these commented out if you're not publishing to s3 from your crates.io instance.

but if I understand the `env()` function correctly it panics if the environment variable is not set. can we make `index_bucket` an `Option<_>` for now, so that we can easily disable it in production if necessary?